doc/modules/multiclass.rst

```diff
@@ -30,21 +30,21 @@ by decomposing such problems into binary classification problems.
 
   - **Multioutput-multiclass classification** and **multi-task classification**
     means that a single estimator has to handle
-    several joint classification tasks. This is a generalization
+    several joint classification tasks. This is both a generalization
     of the multi-label classification task, where the set of classification
-    problem is restricted to binary classification, and of the multi-class
-    classification task. *The output format is a 2d numpy array or sparse
-    matrix.*
+    problems is restricted to binary classification, 
+    as well as a generalization of the multi-class classification task. 
+    *The output format is a 2d numpy array or sparse matrix.*
 
     The set of labels can be different for each output variable.
-    For instance a sample could be assigned "pear" for an output variable that
-    takes possible values in a finite set of species such as "pear", "apple",
-    "orange" and "green" for a second output variable that takes possible values
-    in a finite set of colors such as "green", "red", "orange", "yellow"...
+    For instance, a sample could be assigned "pear" for an output variable that
+    takes possible values in a finite set of species such as "pear" or "apple", 
+    and "blue" or "green" for a second output variable that takes possible values
+    in a finite set of colors such as "green", "red", "blue", "yellow"...
 
     This means that any classifiers handling multi-output
-    multiclass or multi-task classification task
-    supports the multi-label classification task as a special case.
+    multiclass or multi-task classification tasks,
```

Comment:
I think "a multi-output or ... tasks" does not agree in number correctly.
Drop "a".

However, I think "multi-output multiclass or multi-task classification
tasks" is confusing, in part because of the order you say things, but
mostly because I don't think we should be talking about multi-task as if
it's different from MOMC when we're talking about formats rather than
learning. Here is my understanding: MOMC is a format of target that does
not say anything about the relationship between those various outputs; MTC
is a type of learning problem, wherein the multiple outputs are presumed to
have solutions that are mutually informed.

On 14 January 2016 at 16:51, Pieter Arthur de Jong <notifications@github.com
> wrote:

> In doc/modules/multiclass.rst
> <https://github.com/scikit-learn/scikit-learn/pull/5837#discussion_r49690192>
> :
>
> >
> >      This means that any classifiers handling multi-output
> > -    multiclass or multi-task classification task
> > -    supports the multi-label classification task as a special case.
> > +    multiclass or multi-task classification tasks,
>
> I've changed the paragraph to be as follow, lmk whether this makes sense:
>
> "This means that any classifiers handling a multi-output
> multiclass or multi-task classification tasks,
> support the multi-label classification task as a special case.
> Multi-task classification is similar to the multi-output
> classification task with different model formulations. For
> more information, see the relevant estimator documentation."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/5837/files#r49690192>.
>